JASP-Engine/JASP/R/LScommon.R

```diff
@@ -1629,14 +1629,14 @@ hdi.density    <- function(object, credMass=0.95, allowSplit=FALSE, ...) {
              gettextf("Model - for specifying models that will be used for estimation. You can specify the model name, the prior distribution for the parameter %s, and parameters that define the distribution.", parameter_info),
              gettextf("Hypothesis - for setting hypotheses that will be used for testing. You can specify the hypothesis name, the prior probability, the prior distribution for the parameter %s, and parameters that define the distribution.", parameter_info)),
       ifelse(estimation,
-             gettext("Inference - for visualizing the specified models. The individual options show prior, posterior, and prior and posterior distributions. In addition, the multiple models can be shown in one figure (All), with a depth effect (Stacked), or in individual figures (Individual)."),
-             gettext("Inference - for visualizing the specified hypothesis. The individual options show prior distributions, prior predictive distributions, posterior distributions, prior and posterior distributions, and predictive accuracy. In addition, the multiple hypotheses can be shown when considered alone (Conditional), after multiplication by the prior (Joint), and combined together (Marginal)." )),
+             gettext("Inference - for drawing conclusions from the specified models. The options include visualizing prior, posterior, and prior and posterior distributions. In addition, the multiple models can be shown in one figure (All), with a depth effect (Stacked), or in individual figures (Individual)."),
```

Comment:
This sentence sounds weird to me:

`The options include visualizing prior, posterior, and prior and posterior distributions. `

Perhaps change this to 

`The prior and posterior plots can be requested under this section. `

I'm not sure whether the following sentence is relevant already

`In addition, the multiple models can be shown in one figure (All), with a depth effect (Stacked), or in individual figures (Individual).")`

If it is, then perhaps add:

`In addition, the posterior plots of the multiple priors can be shown in one figure (All), with a depth effect (Stacked), or as separate figures (Individual).")`

Reply:
> This sentence sounds weird to me:
> 
> `The options include visualizing prior, posterior, and prior and posterior distributions.`
> 
> Perhaps change this to
> 
> `The prior and posterior plots can be requested under this section.`
> 
> I'm not sure whether the following sentence is relevant already

- I discussed this with EJ and we decided to change it in this way to emphasize that the estimation section includes other options apart from plots (also the estimation summary table). It's possible that I lost a word or two while writing down the notes.

> I'm not sure whether the following sentence is relevant already
> 
> `In addition, the multiple models can be shown in one figure (All), with a depth effect (Stacked), or in individual figures (Individual).")`
> 
> If it is, then perhaps add:
> 
> `In addition, the posterior plots of the multiple priors can be shown in one figure (All), with a depth effect (Stacked), or as separate figures (Individual).")`

Changed to your version, but with models instead of priors

Reply:
I still disagree that `the multiple models are shown in one`. You show posteriors and priors in the figures.

Reply:
Yes, which are part of the models. And I think that it's good the emphasize the difference between the models / hypotheses for the estimation / testing part of the analysis.